### ISSUE TITLE
How to change num_classes when using --weights=last

### ISSUE THREAD

**Issue by MathiasKahlen** [matterport/Mask_RCNN#1513]
So I sucessfully made the balloon example work with multiple classes, however, it seems that I have to train from scratch with imagenet weights if I want to add another class after training the first time. 

1. When I change num_classes and train from imagenet weights it works with whatever number of classes I want

2. When using my own trained weights I get a shape error

Any suggestions?

**Comment by freezurbern** [comment]
I have been experimenting with different numbers of classes and found that I must retrain from the provided coco weights if I change my num_classes. While I do not use imagenet weights, perhaps the idea is the same.

**Comment by MathiasKahlen** [comment]
Yeah I came to the same conclusion. You have to train again if you change the number of classes afterwards :)
